Overview.src.html

```diff
@@ -3944,11 +4021,22 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <span title=concept-request-body>body</span> is non-null, run these substeps:
 
   <ol>
+   <li><p>Let <var>stream</var> be an <span title=concept-empty-ReadableStream>empty</span>
+   <code title=concept-ReadableStream>ReadableStream</code> object.
+
    <li><p>Set <var>input</var>'s <span title=concept-Request-request>request</span>'s
-   <span title=concept-request-body>body</span> to an empty byte stream.
+   <span title=concept-request-body>body</span> to a new <span title=concept-body>body</span> whose
+   <span title=concept-body-stream>stream</span> is <var>stream</var>.
 
-   <li><p>Set <var>input</var>'s
-   <span title=concept-Request-disturbed-flag>disturbed flag</span>.
+   <li>
```

Comment:
Oh, wow, all the way up in step 2! Thanks, that makes sense. I would suggest some slight refactoring then to make this clearer:

- Move step 2 to right before step 31.
- Rename _temporaryBody_ to _rBody_.
- In step 35, rename _stream_ to _dummyStream_.
- Add a note to step 35. Something like:

   > These steps are meant to produce the observable equivalent of "piping" _input_'s [body]'s [stream] into _r_. That is, _r_ is left with a body stream that is disturbed, locked, and empty, while _r_'s body's stream is now equal to what used to be _input_'s.
   >
   > This step will likely change when [`Request`] gets a public `body` property, since we do not want the new [`Request`]'s `body` to be observably equal to _input_'s, and we do not want to observably change _input_'s `body property.

I agree this is hard to spec before we actually have identity transform streams or something similar in place.